src/item.cpp

```diff
@@ -4377,13 +4377,13 @@ bool item::has_flag( const std::string &f ) const
     }
 
     // other item type flags
-    ret = type->item_tags.count( f );
+    ret = type->item_tags.find( f ) != type->item_tags.end();
```

Comment:
This isn't really a performance *fix*. The new code will be just as fast as the existing one. `count` and `find` contain pretty much the same code and the only performance relevant part is the call to `lower_bound` which is done in both functions. The compiler may even generate the exact same output for the existing and for the new code.